templates/settings.yaml.erb

```
@@ -74,6 +74,7 @@
 
 :dynflow:
   :pool_size: <%= scope.lookupvar("foreman::dynflow_pool_size") %>
+  :redis_url: <%= scope.lookupvar("foreman::jobs_sidekiq_redis_url") || "redis://localhost:6379/6" %>
```

This should use the Redis variable:

```suggestion
  :redis_url: <%= scope.lookupvar("foreman::jobs::redis_url") %>
```

You said this fails because the variable is not instantiated at that point which makes me think we should move logic from `foreman::jobs` to `foreman::config`.

An alternative would be to set all the dynflow settings (`pool_size`, `redis_url`) in `worker.yaml` but that might require changes in dynflow itself.

---

If we put it to the `config.pp` we can't use `redis::port` as it is not defined yet.